### PR TITLE
Tag processor

### PR DESCRIPTION
**Pull request by asgerb**
Here we go finally :)

I’ve updated the readme too and removed nokogiri, which was included as a development dependency by mistake.

Oh and I didn’t touch the gem’s version, am guessing this is better to do in the master branch.

### CODE REVIEW

**Comment by tomasc** [comment]
@asgerb seems the tests are not passing :-(

**Comment by asgerb** [comment]
Seems a travis issue: https://travis-ci.org/tomasc/dragonfly_audio/builds/159941408#L327

Passes fine locally at least :)

image

**Comment by asgerb** [comment]
But why is it trying to install mongodb?!

**Comment by asgerb** [comment]
Yay! Thanks!

File: lib/dragonfly_audio/plugin.rb [hunk 1]
```diff
@@ -1,56 +1,31 @@
 require 'dragonfly_audio/analysers/audio_properties'
+require 'dragonfly_audio/processors/tag'
 
 module DragonflyAudio
   class Plugin
-
     def call app, opts={}
       app.add_analyser :audio_properties, DragonflyAudio::Analysers::AudioProperties.new
```
Comment (tomasc) [inline lib/dragonfly_audio/plugin.rb:8]: @asgerb wouldnt it make sense to use the AudioProprties class constants here so we dont list the same props in two places?

File: lib/dragonfly_audio/processors/tag.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+require 'taglib'
+
+# IMPORTANT: See http://robinst.github.io/taglib-ruby/
+
+module DragonflyAudio
+  module Processors
+    class Tag
```
Comment (tomasc) [inline lib/dragonfly_audio/processors/tag.rb:7]: I guess same here, would it be possible to use the constants from AudioProperties classs?

File: test/dragonfly_audio/plugin_test.rb [hunk 3]
```diff
@@ -12,6 +12,30 @@ module DragonflyAudio
       it 'adds #audio_properties' do
         audio.must_respond_to :audio_properties
       end
+
+      %i(
```
Comment (tomasc) [inline test/dragonfly_audio/plugin_test.rb]: Here as well, lets reuse the constants.

**Review by tomasc — COMMENTED** [review]
A few ideas for improvements

**Comment by asgerb** [comment]
Good point, updating!

**Comment by tomasc** [comment]
🎆

**Pull request merged**